app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -691,12 +691,15 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
     }
 #endif
 
+    vrb::Matrix pointerTransformStandalone;
     if (renderMode == device::RenderMode::StandAlone)
-        pointerTransform.TranslateInPlace(kAverageHeight);
+        pointerTransformStandalone = pointerTransform.Translate(kAverageHeight);
+    else
+        pointerTransformStandalone = pointerTransform;
 
-    delegate.SetTransform(mIndex, pointerTransform);
+    delegate.SetTransform(mIndex, vrb::Matrix::Identity());
     delegate.SetImmersiveBeamTransform(mIndex, pointerTransform);
-    delegate.SetBeamTransform(mIndex, vrb::Matrix::Identity());
+    delegate.SetBeamTransform(mIndex, pointerTransformStandalone);
```

Comment:
This change looks weird, why are we doing it? I mean, I'd be surprised if it does not completely break the current builds

Also the if-else change seems not needed, it's basically copying a matrix for no apparent reason (and converting a translateinplace into a translate)

Reply:
> Also, notice that this is a new variable pointerTransformStandalone. I added it because we need pointerTransform without any translation too.

I think I checked the rest of the method and it was not used...

Reply:
> The problem is that this logic was already buggy since I introduced it. The `transformMatrix` controller member is a transform specific to the controller, independently of any active pose. During hand-tracking, we were abusing it by storing the pointer transform; and it only worked because we were setting `immersiveTransform` to the identity matrix. This problem became apparent when WebXR hand-tracking was introduced.
> 
> After a lot of testing and debugging, the changes in this patch brings back order to this functionality and now hand-tracking works fine in Wolvic UI, in WebXR emulating hand-tracking, and in WebXR hand-tracking proper. It doesn't bring any regression as far as I can tell.

if that's the case then let's move this change to a separate commit with an explanation of why it is wrong and what is fixing.